.changeset/gentle-suits-scream.md

```diff
@@ -0,0 +1,26 @@
+---
+"@paprika/uploader": major
+---
+
+- Replaced `defaultIsDisabled` prop by `isDisabled` prop
+- Added `zIndex` prop
+
+#### why the change was made:
+
+`defaultIsDisabled` can't be changed once mount.
```

Comment:
Does this imply that `isDisabled` is now a controlled prop, and can be changed at any time by the consumer?

Reply:
Right